### PR TITLE
chore: improve README

### PR DESCRIPTION
**Pull request by drinchev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,52 @@
...
 
-Run development server on port 5050.
+## Features
 
-```
-npm run start
-```
-
-## Running the tests
+The Lightelligence React are a set of React components, styled according to our
```
Comment (JonathanStoye) [inline README.md]:
```suggestion
Lightelligence React is a set of React components, styled according to our
```

File: README.md [hunk 2]
```diff
@@ -1,44 +1,52 @@
...
+design system. It is a representation of
```
Comment (JonathanStoye) [inline README.md]:
```suggestion
design system. It is a React.js implementation of
```

File: README.md [hunk 3]
```diff
@@ -1,44 +1,52 @@
...
-- Code formatting: [prettier](https://github.com/prettier/prettier).
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '@lightelligence/react/dist/index.css';
+import { Button, COLOR_PRIMARY } from '@lightelligence/react';
 
-Please, consider installing eslint and prettier plugins for your IDE.
+const App = () => (
+  <Frame>
```
Comment (JonathanStoye) [inline README.md]: `Frame` is not defined 😛

Comment (drinchev) [reply README.md]: Nice catch.

File: README.md [hunk 4]
```diff
@@ -1,44 +1,52 @@
...
+wrapping your content in [`<Frame />`](https://lightelligence-io.github.io/react/#/Components/Frame)
+component.
 
-- JavaScript/React: [airbnb coding guidelines](https://www.npmjs.com/package/eslint-config-airbnb).
-- Code formatting: [prettier](https://github.com/prettier/prettier).
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import '@lightelligence/react/dist/index.css';
+import { Button, COLOR_PRIMARY } from '@lightelligence/react';
```
Comment (JonathanStoye) [inline README.md]:
```suggestion
import { Button, Frame, COLOR_PRIMARY } from '@lightelligence/react';
```

**Pull request merged**